src/styles/variables/variables.scss

```
@@ -0,0 +1,6 @@
+$font-family: Roboto, sans-serif;
+$text-color: #060b35;
+$secondary-color: #616070;
+$button-main-backgroud: #00acdc;
+$button-border-text-active: #00acdc;
```

Why did you create 2 variables with same values?
```suggestion
$button-main-backgroud: #00acdc;
```